tests/integration/jwt-authorizer/handler.js

```diff
@@ -0,0 +1,16 @@
+'use strict'
+
+const { stringify } = JSON
```

Comment:
Destructuring `stringify` seems a little unconventional

Reply:
it does... hmm. That's what I get for copy pasta from the authorizer tests :)